index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta property="og:image" content="https://i.postimg.cc/W17TS4qW/project-desktop-v.jpg">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css">
+    <link rel="stylesheet" href="./style.css">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+        crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+        type="text/css" />
+    <title>Microverse Capstone One</title>
+</head>
+
+<body>
+    <header>
+        <nav class="header-menu">
+            <a href="#" class="mx-3 text-white"><i class="fab fa-facebook"></i></a>
+            <a href="#" class="mx-3 text-white"><i class="fab fa-twitter"></i></a>
+            <a href="#" class="mx-3 text-white">English</a>
+            <a href="#" class="mx-3 text-white">MyPage</a>
+            <a href="#" class="mx-3 text-white">Logout</a>
+        </nav>
+        <nav class="desktop-menu">
```

Comment:
- [ ] Please add some margin or padding to the right side of the top-navbar, so that it matches the design. In the design, the top navbar and the bottom navlinks are not aligned with each other, and they have some space on the right edge.

Cindy's design
----
<img width="598" alt="Screenshot 2023-07-07 at 8 41 35 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/4dab2285-10bc-4b2e-9905-698dae2023bc">

Your design
----
<img width="1512" alt="Screenshot 2023-07-07 at 6 26 15 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/3acddb02-731e-420a-b63c-924317ae9b91">